apps/mobile/src/components/widgets/collectibles/collectibles.layout.tsx

```diff
@@ -0,0 +1,81 @@
+import { NonFungibleCryptoAssetInfo } from '@leather.io/models';
+import { Box, CollectibleCard } from '@leather.io/ui/native';
+import { isDefined, isEmptyString } from '@leather.io/utils';
+
+import { CollectibleCaption } from './collectible-caption';
+
+interface CollectiblesLayoutProps {
+  collectibles: NonFungibleCryptoAssetInfo[];
+  view?: 'widget' | 'gallery';
+}
+
+function filterCollectibles(collectible: NonFungibleCryptoAssetInfo) {
+  switch (collectible.protocol) {
+    case 'inscription':
+      return !isEmptyString(collectible.src) && isDefined(collectible.src);
+    case 'sip9':
+      return (
+        !isEmptyString(collectible.cachedImage) &&
+        isDefined(collectible.cachedImage) &&
+        // BNS - Archive is not a valid collectible
```

Comment:
I added filtering out collectibles with the name `BNS - Archive` as I'm not sure what that is. I guess BNS names? 

I cross checked the `extension` and BNS names seem to be loaded a different way in `StacksBnsName` in `StacksCryptoAssets` so I need to do more work there